tests/test_edgeql_volatility.py

```diff
@@ -427,6 +427,19 @@ async def test_edgeql_volatility_with_04(self):
             [False, False, True, True],
         )
 
+    async def test_edgeql_volatility_with_05(self):
+        await self.con.execute(r'''
+            CREATE TYPE Foo { CREATE PROPERTY asdf -> tuple<float64> };
+        ''')
+
+        await self.con.query(r'''
+            WITH X := (random(),) SELECT X.0;
+        ''')
+
+        await self.con.query(r'''
+            WITH X := {(random(),),(random(),)} SELECT X.0;
+        ''')
```

Comment:
Since this test doesn't check the output, I assume it tests that the compiler can compile this query?

Reply:
Yeah, and the result is a random number so there's not a whole lot of testing we can do on it